zenoh-plugin-mqtt/src/lib.rs

```diff
@@ -41,32 +42,33 @@ macro_rules! ke_for_sure {
     };
 }
 
-pub const GIT_VERSION: &str = git_version!(prefix = "v", cargo_prefix = "v");
 lazy_static::lazy_static! {
-    pub static ref LONG_VERSION: String = format!("{} built with {}", GIT_VERSION, env!("RUSTC_VERSION"));
     static ref KE_PREFIX_ADMIN_SPACE: &'static keyexpr = ke_for_sure!("@/service");
     static ref ADMIN_SPACE_KE_VERSION: &'static keyexpr = ke_for_sure!("version");
     static ref ADMIN_SPACE_KE_CONFIG: &'static keyexpr = ke_for_sure!("config");
 }
 
+#[cfg(feature = "no_mangle")]
```

Comment:
Is it required, as it was not added in https://github.com/eclipse-zenoh/zenoh-plugin-dds and it seems to work ?
If not, I think it's also useless to keep this feature in Cargo.toml

Reply:
I had compilation problem without it, but now can't reproduce the issue. Created issue to further investigate is it really needed: https://github.com/eclipse-zenoh/zenoh/issues/751